content/docs/user-guide/gems/reference/environment/vegetation.md

```diff
@@ -0,0 +1,207 @@
+|     | **Position Max Y** | Override value for the maximum position offset on the Y axis. | Float |
+|     | **Position Min Z** | Override value for the minimum position offset on the Z axis. | Float |
+|     | **Position Max Z** | Override value for the maximum position offset on the Z axis. | Float |
+| Rotation |     | Settings for controlling per instance overrides for the rotation modifier component. |     |
+|     | **Rotation Override Enabled** | When enabled, the rotation modifier component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Rotation Min X** | Override value for the minimum rotation offset on the X axis. | Float |
+|     | **Rotation Max X** | Override value for the maximum rotation offset on the X axis. | Float |
+|     | **Rotation Min Y** | Override value for the minimum rotation offset on the Y axis. | Float |
+|     | **Rotation Max Y** | Override value for the maximum rotation offset on the Y axis. | Float |
+|     | **Rotation Min Z** | Override value for the minimum rotation offset on the Z axis. | Float |
+|     | **Rotation Max Z** | Override value for the maximum rotation offset on the Z axis. | Float |
+| Scale |     | Settings for controlling per instance overrides for the scale modifier component. |     |
+|     | **Scale Override Enabled** | When enabled, the scale modifier component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Scale Min** | Override value for the minimum scale multiplier. | Float |
+|     | **Scale Max** | Override value for the maximum scale multiplier. | Float |
+| Altitude Filter |     | Settings for controlling per instance overrides for the altitude filter component. |     |
+|     | **Altitude Filter Override Enabled** | When enabled, the altitude filter component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Altitude Filter Min** | Override value for the minimum altitude accepted by the filter. | Float |
+|     | **Altitude Filter Max** | Override value for the maximum altitude accepted by the filter. | Float |
+| Distance Between Filter (Radius) |     | Settings for controlling per instance overrides for the distance between filter component. |     |
+|     | **Distance Between Filter Override Enabled** | When enabled, the distance between filter component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Bound Mode** | When performing distance checks between two instances, this setting determines whether the radius of the mesh asset is used vs a manually entered radius<br><br>Radius<br><br>MeshRadius. |     |
+|     | **Radius Min** | User defined radius for distance checks. | Float |
+| Surface Slope Alignment |     |     |     |
```

Comment:
```suggestion
| Surface Slope Alignment |     |     |     |
```
This and the rest of the component overrides don't have a property group description.  I don't think in any case it is a major issue, we could unify their treatment it at some point (or not). I have left property group descriptions blank like this in many other components.

Reply:
I actually want to reformat this so it matches the standard "properties" formatting. With that format, the group names will be formatted as headers, instead of having a column like this.